raml-lint-check/lint-check-rules/src/test/java/uk/gov/justice/services/raml/lintcheck/it/ActionsHaveHandlersMojoTest.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.justice.services.raml.lintcheck.it;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import uk.gov.justice.raml.maven.lintchecker.LintCheckMojo;
+import uk.gov.justice.raml.maven.test.utils.BetterAbstractMojoTestCase;
+
+import java.io.File;
+
+import org.apache.maven.plugin.MojoExecutionException;
+
+public class ActionsHaveHandlersMojoTest extends BetterAbstractMojoTestCase {
+
+    public void testShouldFailTest() throws Exception {
+
+        final File pom = getTestFile( "src/test/resources/it/pom.xml" );
+        assertNotNull( pom );
+        assertTrue( pom.exists() );
+
+        final LintCheckMojo myMojo = (LintCheckMojo) lookupConfiguredMojo(pom, "lint-check");
+
+        assertNotNull( myMojo );
+
+        try {
+            myMojo.execute();
+            fail();
+        } catch (MojoExecutionException e) {
```

Comment:
Could use the JUnit
    @Rule
    public ExpectedException expectedException = ExpectedException.none();